index.rst

```diff
@@ -329,11 +329,18 @@ What's new in PyTorch tutorials?
    :tags: Production,TorchScript
 
 .. customcarditem::
-   :header: (optional) Exporting a Model from PyTorch to ONNX and Running it using ONNX Runtime
+   :header: (optional) Exporting a PyTorch model to ONNX using TorchDynamo backend and Running it using ONNX Runtime
+   :card_description: Build a image classifier model in PyTorch and convert it to ONNX before deploying it with ONNX Runtime.
+   :image: _static/img/thumbnails/cropped/Exporting-PyTorch-Models-to-ONNX-Graphs.png
+   :link: beginner/export_simple_model_to_onnx_tutorial.html
+   :tags: ONNX
+
+.. customcarditem::
+   :header: (optional) Exporting a PyTorch Model to ONNX using TorchScript backend and Running it using ONNX Runtime
    :card_description:  Convert a model defined in PyTorch into the ONNX format and then run it with ONNX Runtime.
    :image: _static/img/thumbnails/cropped/optional-Exporting-a-Model-from-PyTorch-to-ONNX-and-Running-it-using-ONNX-Runtime.png
    :link: advanced/super_resolution_with_onnxruntime.html
-   :tags: Production
+   :tags: Production,ONNX
```

Comment:
Please change the tag to describe the functionality more than the name of the project. I suggest HW Optimization or similar.

Reply:
Can we keep both `Production` and `ONNX` tags?

For pytorch there are different types of `Production` deployments - this tutorial lives inside the `Deploying PyTorch Models in Production` category (left menu). But ONNX Runtime users only care about the `ONNX` backend, so having this flag helps them finding all beginner, intermediate or advanced tutorials directly through the tag cloud

Reply:
Discussing this point here https://github.com/pytorch/tutorials/pull/2550#issuecomment-1711964329